frontend/packages/topology/src/components/modals/MoveConnectionModal.tsx

```diff
@@ -98,21 +93,30 @@ const MoveConnectionForm: React.FC<
         </Title>
         <div className="pf-v5-c-form">
           <FormGroup fieldId="target-node" label="Target">
-            <DropdownDeprecated
+            <Select
               id="target-node-dropdown"
               className="dropdown--full-width"
-              toggle={
-                <DropdownToggleDeprecated
-                  id="toggle-id"
-                  onToggle={onToggle}
-                  toggleIndicator={CaretDownIcon}
-                >
-                  {nodeItem(values.target)}
-                </DropdownToggleDeprecated>
-              }
+              onSelect={(_, value: Node) => {
+                if (value) {
+                  values.target = value;
+                }
+                setOpen(false);
+              }}
+              toggle={toggle}
```

Comment:
Need to add prop `onOpenChange={(open) => setOpen(open)}` to close the options menu when clicking anywhere in the window.

Reply:
amended, thanks for catching :)